x/axelarcork/types/genesis.go

```diff
@@ -0,0 +1,33 @@
+package types
+
+const DefaultParamspace = ModuleName
+
+// DefaultGenesisState get raw genesis raw message for testing
+func DefaultGenesisState() GenesisState {
+	defaultParams := DefaultParams()
+	return GenesisState{
+		Params:              &defaultParams,
+		ChainConfigurations: ChainConfigurations{},
+		InvalidationNonces:  []uint64{},
+		CellarIds:           []*CellarIDSet{},
+		ScheduledCorks:      []*ScheduledCorks{},
+		CorkResults:         []*CorkResults{},
+	}
+}
+
+// Validate performs a basic stateless validation of the genesis fields.
+func (gs GenesisState) Validate() error {
+	if err := gs.Params.ValidateBasic(); err != nil {
+		return err
+	}
+
+	for _, cc := range gs.ChainConfigurations.Configurations {
+		if err := cc.ValidateBasic(); err != nil {
+			return err
+		}
+	}
+
+	// todo (mvid): flesh out initial validation
```

Comment:
Is this still a TODO?

Reply:
I think we've defaulted to implementing basic validation for most of our custom types. You'd want to be able to validate members of submitted messages when one of them is one of these types.